spec/system/referrals/user_adds_organisation_details_spec.rb

```diff
@@ -0,0 +1,93 @@
+require "rails_helper"
+
+RSpec.describe "Employer Referral: Organisation", type: :system do
+  scenario "User provides the organisation details" do
+    given_the_service_is_open
+    and_i_am_signed_in
+    and_the_employer_form_feature_is_active
+    and_i_have_an_existing_referral
+    and_i_am_on_the_referral_summary_page
+    when_i_click_on_your_organisation
+    then_i_am_on_the_organisation_details_page
+
+    when_i_click_save_and_continue
+    then_i_am_asked_to_make_a_choice
+
+    when_i_choose_no_come_back_later
+    and_i_click_save_and_continue
+    then_i_am_on_the_referral_summary_page
+    and_i_see_your_organisation_flagged_as_incomplete
+
+    when_i_click_on_your_organisation
+    and_i_choose_complete
+    and_i_click_save_and_continue
+    then_i_am_on_the_referral_summary_page
+    and_i_see_your_organisation_flagged_as_complete
+  end
+
+  private
+
+  def and_i_choose_complete
+    choose "Yes, I’ve completed this section", visible: false
+  end
+
+  def and_i_see_your_organisation_flagged_as_incomplete
+    your_organisation_row =
+      find(".app-task-list__item", text: "Your organisation")
+    expect(your_organisation_row).to have_content("INCOMPLETE")
+  end
+
+  def given_the_service_is_open
+    FeatureFlags::FeatureFlag.activate(:service_open)
+  end
+
+  def and_i_am_signed_in
+    user = create(:user)
+    sign_in(user)
```

Comment:
```suggestion
    @user = create(:user)
    sign_in(@user)
```

(paired with the change below)